1.Exploratory_Data_Analysis/0.nomic_cleanup.ipynb

```diff
@@ -187,25 +187,50 @@
     "nomic_df.head(3)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    nomic_df["Metadata_Treatment"] = np.select(condlist=conditions, choicelist=results)
</pre><p>Interesting way to do this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>